DataCapturing/Model/GeoLocationMO+CoreDataProperties.swift

```diff
@@ -2,13 +2,14 @@
 //  GeoLocationMO+CoreDataProperties.swift
```

Comment:
This whole class is generated by Xcode. You can safely ignore it. The code is not documented and does not follow the usual coding standards, since on the next change all manual changes are overwritten.